readthedocs/settings/base.py

```diff
@@ -565,7 +565,7 @@ def TEMPLATES(self):
         },
     }
     # Always point to the latest stable release.
-    RTD_DOCKER_BUILD_SETTINGS['tools']['python']['3'] = RTD_DOCKER_BUILD_SETTINGS['tools']['python']['3.9']
+    RTD_DOCKER_BUILD_SETTINGS['tools']['python']['3'] = RTD_DOCKER_BUILD_SETTINGS['tools']['python']['3.10']
```

Comment:
Should we wait a little bit to see if any bugs fall out before making this default? That will upgrade a decent number of people I'm guessing, and I don't know if the ecosystem is ready for it.

Reply:
This will only affect only people using `build.tools.python: 3` --which at this moment, there isn't anyone probably.